### PR TITLE
 Allow for packaging of only a small set of wordlists.

### PR DESCRIPTION
**Pull request by junderw**
This seems like a simple way of doing it that doesn't break it.

Thoughts?

### CODE REVIEW

File: wordlists.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+// browserify by default only pulls in files that are hard coded in requires
+// In order of last to first in this file, the default wordlist will be chosen
+// based on what is present. (Bundles may remove wordlists they don't need)
+var wordlists = {}
+var wordlist
+try {
+  wordlist = require('./wordlists/chinese_simplified.json')
+  wordlists.chinese_simplified = wordlist
```
Comment (fanatid) [inline wordlists.js]: Out of interest, why not as `wordlists.chinese_simplified = require('./wordlists/chinese_simplified.json')` ?

Comment (junderw) [reply wordlists.js]: to get a default wordlist.

Comment (fanatid) [reply wordlists.js]: ah, sorry, I missed this in `exports`.

**Comment by zyuhel** [comment]
Wouldn't it be better to make getting default wordlist a separate function so there won't be such duplicate parts?
`
  wordlist = wordlist || DEFAULT_WORDLIST
  if (!wordlist) {
    throw new Error(WORDLIST_REQUIRED)
  }
`

**Comment by junderw** [comment]
Closed in favor of #104

**Pull request closed without merging**